src/System.Console/src/System.Console.builds

```diff
@@ -9,23 +9,8 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Console.csproj">
-      <OSGroup>Unix</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.3</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
-      <TargetGroup>net46</TargetGroup>
-    </Project>
-    <Project Include="System.Console.csproj">
       <TargetGroup>net463</TargetGroup>
     </Project>
-    <Project Include="System.Console.csproj">
```

Comment:
Actually I think @ianhays's changes are in the release/1.1.0 branch and once they ship stable we can harvest it from that new package but until then we need this build back.